06-reactjs/components/movie-row.js

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
+
+var MovieRow = React.createClass( {
+   propTypes: function () {
+       title: React.PropTypes.string;
```

Comment:
This is not right, since you are using this.props.movie[...] instead of this.props[...]